src/minigames/final/typingMinigame.ts

```diff
@@ -0,0 +1,191 @@
+import { Trie, TrieNode } from "@datastructures-js/trie";
+import { KeyboardMinigame } from "../keyboardMinigame";
+import { Assets, Sprite } from "pixi.js";
+import { MINIGAME_ASSET_ALIASES } from "../assets";
+
+const TARGET_SENTENCES = [
+  "the pain you feel today is the strength you feel tomorrow",
+  "no pain no gain no train no muscle #power #grind #alpha",
+  "they told me i could not do it they said i would fail i ignored them and kept going #bulk #swole",
```

Comment:
Can we avoid 1 letter words? There's a section below that introduces bugs for 1 letter words (see line 117). Alternatively we could remove that QoL improvement?